.github/workflows/sycl-nightly.yml

```
@@ -23,7 +23,7 @@ jobs:
       # prefer widespread gzip compression.
       artifact_archive_name: sycl_linux.tar.gz
 
-  ubuntu2204_shared_build:
+  ubuntu2404_shared_build:
```

```suggestion
  linux_shared_build:
```
The job doesn't specify OS. It just re-uses sycl-linux-build.yml workflow. Let's not hard code the name of the Linux flavor in the name.

---

I'd personally prefer explicitly specifying the OS in the job name as we have multiple build jobs in Nightly, with different OS versions. 
I don't have strong opinion on it, so if other folks from @intel/dpcpp-devops-reviewers feel the same, I'd be happy to make the change.

---

AFAIK, SYCL implementation does not depend on Linux OS flavor. There are differences between Linux and Windows. However, if the only difference between the two systems is the Linux OS version, the results must be the same.

SYCL implementation depends on GNU toolchain, CUDA SDK, etc. Having separate jobs for different versions of these dependencies make sense (e.g. Ubuntu "default" GNU, CUDA, etc. vs RHEL "default" ...). AFAIK, we don't do that in existing CI.

---

@uditagarwal97, if I understand Andrei correctly, he suggests keeping the OS version name in the job name. @aelovikov-intel, did I get it right?

---

No, I meant either 
* keep using default image of the `sycl-linux-build.yml` and change the job name (which Udit did)
* keep the name, but spell out image name explicitly when invoking `sycl-linux-build.yml`